.github/workflows/coverage.yml

```
@@ -3,7 +3,11 @@ name: Coverage
 on:
   push:
     branches:
+      - dev
       - stable
+    paths:
+      - '**.rs'
+      - '**.toml'
```

```suggestion
      - '**.toml'
      - '.github/workflows/coverage.yml'
```

You might want this workflow to run when this file is changed too

---

Good idea 👍